core/commands/keystore.go

```diff
@@ -229,6 +230,7 @@ var keyImportCmd = &cmds.Command{
 		cmds.StringArg("name", true, false, "name to associate with key in keychain"),
 		cmds.FileArg("key", true, false, "key provided by generate or export"),
 	},
+	PreRun: daemonNotRunning,
```

Comment:
Import can be run while the daemon is running (give it a shot, it works). Although it's concerning that breaking this didn't cause our tests to fail. We should probably add online/daemon sharness tests to the keystore tests (i.e. verify import works, export fails, rotate fails).

Reply:
Fixed

Reply:
Added tests to check import, export, and rotate while daemon running.